site/content/en/docs/contribution_guidelines/_index.md

```diff
@@ -12,12 +12,15 @@ _As contributors and maintainers of this project, and in the interest of fosteri
 
 ## Getting Started
 
-We have full documentation on how to get started contributing here:
+We have full documentation on how to get started contributing [here](./development)!
```

Comment:
Shall we have an absolute link here, since the link doesn't work from the GitHub ?
<img width="1269" height="303" alt="Screenshot 2025-07-31 at 23 07 25" src="https://github.com/user-attachments/assets/5cf272df-4f29-4ac1-a5cc-13b176a5326e" />